packages/integration-tests/rules.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const rulesApi = require('@cumulus/api-client/rules');
+const { randomId } = require('@cumulus/common/test-utils');
+
+const buildRule = (overrides = {}) => ({
+  name: randomId('rule_'),
+  ...overrides
+});
+
+const buildOneTimeRule = (overrides = {}) =>
+  buildRule({
+    ...overrides,
+    rule: {
+      type: 'onetime'
+    }
+  });
+
+/**
+ * Build a onetime rule and create it using the Cumulus API
+ *
+ * See the `@cumulus/integration-tests` README for more information
+ *
+ * @param {string} prefix - the Cumulus stack name
+ * @param {Object} overrides - properties to set on the rule, overriding the
+ *   defaults
+ * @returns {Promise<Object>} the generated rule
+ */
+const createOneTimeRule = async (prefix, overrides = {}) => {
```

Comment:
Why create a function specific to creating a "one-time" rule? Then you're going to have to create separate ones for SQS, Kinesis, etc? Why not have the type be an argument?